### ISSUE TITLE
not capturing console logs if we use any swift POD in the project

### ISSUE THREAD

**Issue by Duraiamuthan** [Naituw/WBWebViewConsole#5]
_How to reproduce the problem ?_

The problem is with the POD WBWebViewConsole

Run the [attached project](https://drive.google.com/open?id=0B-yFxDqROZVpeC05V2x0YXo1dzg) in any iOS 8+ devices that has internet connection as it loads a html doc from google drive.

in the html doc … click General/Info/Warning/Debug/Error Log

[![enter image description here](http://i.stack.imgur.com/sGW1f.jpg)](http://i.stack.imgur.com/sGW1f.jpg)

you will some text appearing on the html page whenever u click any of the above buttons….

To me the library is about capturing the logs that’s generated in html page and thats my use case

whenever u click the button apart from showing some text in html page I am writing some log in background.

[![enter image description here](http://i.stack.imgur.com/GFIJg.png)](http://i.stack.imgur.com/GFIJg.png)

now click on the button **get logs** … and see the logs in Xcode IDE… you will see the all the console logs that’s generated in html 

[![enter image description here](http://i.stack.imgur.com/gjm1W.png)](http://i.stack.imgur.com/gjm1W.png)

Get logs-> is a native button super imposed on webview..the library lets us read the console logs from wkwebview 
..

Now in the POD 
u uncomment the following lines

```
use_frameworks!
pod 'PLCrashReporter'
pod 'XWebView', '~> 0.9.5’
```

and do pod install

then paste the following code on top of ViewController

```
import WBWebViewConsole
```

now u run the project on device and click some buttons in html page and click _get logs_ you can't see comments that's generated in the html page

_**So what Exactly is the problem:**_
The below two delegates would get called  in _WBWKWebView_ if any log is wrriten by JS but its not happening

```
- (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler
- (void)wb_evaluateJavaScript:(NSString *)javaScriptString completionHandler:(void (^)(NSString *, NSError *))completionHandler
```

But the same delegate which is below gets fired while loading a url 

`- (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler
`

Herewith is the [project](https://drive.google.com/open?id=0B-yFxDqROZVpOUFaWkFoQ2hfWUE) that faces this problem

But while creating the instance of WBWKWebView and loading a URL all the necessary delegates of this class are getting fired.

The above delegates are supposed to fire when a log is written while clicking the button in html page

_Note:_

If we use swift pod then we have to add it as frameworks only ...we can't just add a single pod as framework every pod will be added as framework only.

**Comment by Duraiamuthan** [comment]
I have raised a question in stack overflow as well !

**Comment by KumaraKL** [comment]
The problem is with the search paths under build settings, which are overriding the other libs paths . Just install one by one the pod .. And it works fine .

**Comment by Duraiamuthan** [comment]
Buddy Thanks...But its not working .. I have a doubt that this library won't work if its compiled as Dynamic Framework... if we use any swift lib in pod all the libs in the POD will become Dynamic framework.... I am still looking for solution ...

**Comment by Naituw** [comment]
Thank you for the investigate, Duraiamuthan

When JSBridge init, it will add some "UserScripts" to webview, which source is loaded from `[NSBundle mainBundle]` in previous version. But if it's in a framework, the resource files is in the framework bundle instead of mainBundle.

This is actually been fixed in the master branch year ago (uses [NSBundle bundleForClass:]), but NOT in the release version. 

So I just released version 1.0.2, you can:
1. pod update
2. restart Xcode and clean your project
3. build and run again

Reopen this issue if you still encounter any problem.

**Comment by Duraiamuthan** [comment]
Pleasure is mine 👍  Thanks for the fix 💯  ! I really appreciate your efforts on making the library tremendously awesome 👍  ! You are the Rock Star  !
